srcpkgs/refind/files/kernel.post-install

```diff
@@ -13,7 +13,7 @@ if [ "z$UPDATE_REFIND_CONF" != "z1" ]; then
 fi
 
 # Default refind.conf
-: "${REFIND_CONF:=/boot/EFI/refind/refind.conf}"
+: "${REFIND_CONF:=boot/EFI/refind/refind.conf}"
```

Comment:
This is not correct.
It'll break if user has changed their configuration.
And this will break the path relative to <esp> mountpoint.

Maybe something like this?

```diff
diff --git a/srcpkgs/refind/files/kernel.post-install b/srcpkgs/refind/files/kernel.post-install
index 88b261aaac..70f26936c0 100755
--- a/srcpkgs/refind/files/kernel.post-install
+++ b/srcpkgs/refind/files/kernel.post-install
@@ -15,7 +15,14 @@ fi
 # Default refind.conf
 : "${REFIND_CONF:=/boot/EFI/refind/refind.conf}"
 
+abs_rootdir="$(cd "${ROOTDIR}" && pwd)"
+if [ "$abs_rootdir" != "/" ]; then
+	abs_rootdir="$abs_rootdir/"
+fi
+
+REFIND_CONF="${abs_rootdir}${REFIND_CONF#/}"
 zrefind_dir="${REFIND_CONF%/*}"
+
 mkdir -p "$zrefind_dir"
 touch "$REFIND_CONF"
 
diff --git a/srcpkgs/refind/files/kernel.post-remove b/srcpkgs/refind/files/kernel.post-remove
index efb3786396..b50f19680b 100755
--- a/srcpkgs/refind/files/kernel.post-remove
+++ b/srcpkgs/refind/files/kernel.post-remove
@@ -13,7 +13,8 @@ if [ "z$UPDATE_REFIND_CONF" != "z1" ]; then
 fi
 
 # Default refind.conf
-: "${REFIND_CONF:=/boot/EFI/refind/refind.conf}"
+: "${REFIND_CONF:=boot/EFI/refind/refind.conf}"
+REFIND_CONF="${REFIND_CONF#/}"
 
 [ -f "$REFIND_CONF" ] || exit 0
 
```

Reply:
The `post-install` is too complicated. What about
```
REFIND_CONF="$(readlink -m "${ROOTDIR}/${REFIND_CONF:-boot/EFI/refind/refind.conf}" 2>/dev/null)"
[ -n "${REFIND_CONF}" ] || exit 0
```
Since we already rely on `cat`, which is in `coreutils` alongside `readlink`, I don't have a problem just assuming readlink exists. We also just assume `awk` and `sed` are available anyway.